sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/reflect/DoFnSignatures.java

```diff
@@ -151,57 +173,49 @@ private static DoFnSignature parseSignature(Class<? extends DoFn> fnClass) {
     if (params.length > 0) {
       contextToken = fnClass.resolveType(params[0]);
     }
-    checkArgument(
+    errors.checkArgument(
         contextToken != null && contextToken.equals(processContextToken),
-        "%s must take a %s as its first argument",
-        format(m),
+        "Must take %s as its first argument",
```

Comment:
Should this read "Must take %s as the first argument" since the subject of the sentence is no longer present?